data_prep/tcga_data_prep.rmd

```diff
@@ -401,7 +403,7 @@ saveRDS(skcm, "projects/SKCM.rds")
 
 ```{r}
 
-# This part is separated from this Rmd file because of the conflict issues based on specific packages, please check the "Zero_Count_Filtration_Step" Rmd file.
+# This part is separated from this Rmd file because of the conflict issues based on specific packages, please check the "zero_count_filtration_step" Rmd file.
```

Comment:
This sounds confusing and not descriptive. Instead, use the following:

"Removal of lowly expressed genes are performed elsewhere. Please seethe 'zero_count_filtration_step.R' code."